addon-test-support/@ember/test-helpers/index.ts

```diff
@@ -49,10 +49,13 @@ export { getDebugInfo } from './-internal/debug-info';
 export { default as registerDebugInfoHelper } from './-internal/debug-info-helpers';
 export type { TestMetadata } from './test-metadata';
 export { default as getTestMetadata } from './test-metadata';
-export {
-  registerHook as _registerHook,
-  runHooks as _runHooks,
-} from './-internal/helper-hooks';
+export { registerHook, runHooks } from './helper-hooks';
+export type {
+  Hook,
+  HookHelperName,
+  HookLabel,
+  HookUnregister,
+} from './helper-hooks';
```

Comment:
We should be able to fold these together as well as dropping the now-removed type exports for `HookHelperName` and `HookLabel`:

```suggestion
export {
  registerHook,
  runHooks,
  type Hook,
  type HookUnregister,
} from './helper-hooks';
```